_pkgdown.yml

```diff
@@ -4,11 +4,14 @@ template:
 navbar:
   structure:
     left: []
-    right: [intro, packages, methodology, reference, articles, tutorials, news, github, search]
+    right: [intro, packages, methodology, reference, tutorials, news, github, search]
```

Comment:
FYI intro, tutorials, and news also seem unused.

Reply:
These were intentionally left in because they're common/default/normal things to have in a pkgdown site, and the goal of this line is to define the structure/order of these items *if* they exist, so if one of those things is added in the future it will automatically show up in the menu and be placed properly in line with how our other pkgdown sites are structured.

On that note, does this PR mean that we can no longer use articles in the future because of this change? i.e. we have to choose between having this "Global search" function and having the ability to use articles?